### ISSUE TITLE
Improve look and feel of project metadata dialog

### ISSUE THREAD

**Issue by wetneb** [OpenRefine/OpenRefine#5701]
The project metadata panel (which can be obtained by clicking one of the "About" links in the project list) is not very inviting:

![image](https://user-images.githubusercontent.com/309908/225703191-353527e2-8e56-41ef-8a47-253190495ed7.png)

Therefore I would propose that we:
* [ ] identifiy in which ways this dialog is departing from the style of the rest of the UI
* [ ] propose any other sensible design changes which would make this panel easier to work with and more inviting
* [ ] implement those changes

**Comment by Abbe98** [comment]
Pinging in myself as I intend to harmonize table styles as a part of #3017. Happy to implement other improvements when I'm spending time in this part of the codebase as well.

**Comment by tfmorris** [comment]
The entire project metadata facility feels only half thought through and barely implemented. I would extend the design effort to include the project listing page. There's no reason I shouldn't be able to select which columns are displayed and in what order. If I want to sort projects by their Creation Date, I should be able to do that.

Is there documentation somewhere about what each of these fields means to help guide the designer?

**Comment by wetneb** [comment]
I don't think there is any documentation but I generally share your dissatisfaction about the project list. For instance, for me, the fields such as "Creator", "Subject" and "Description" are useless and should be removed, or at least not present by default.

**Comment by Abbe98** [comment]
> I don't think there is any documentation but I generally share your dissatisfaction about the project list. For instance, for me, the fields such as "Creator", "Subject" and "Description" are useless and should be removed, or at least not present by default.

I can see why one would remove these fields by default given they aren't that accessible, we are, however, very dependent on these being displayed. For us "Description" and "Creator" are essentially holding provenience and job summaries from our ETL orchestration platform:

![Screenshot 2023-03-17 at 05-51-56 OpenRefine](https://user-images.githubusercontent.com/2631719/225815758-94e69c2e-2745-4e5c-bdfb-e0ecefd754e3.png)

I'm, however, very much for making the various columns optional/configurable!

**Comment by tfmorris** [comment]
> For us "Description" and "Creator" are essentially holding provenience and job summaries from our ETL orchestration platform:

What is "our ETL orchestration platform" in this context?

**Comment by Abbe98** [comment]
@tfmorris it's a Dagster instance and a RunDeck instance.

https://dagster.io/
https://www.rundeck.com/

**Comment by Lydiaofficial** [comment]
@wetneb could this be assigned to me, please?

**Comment by Lydiaofficial** [comment]
Having used the Openrefine app for almost 3 weeks now and gotten somewhat familiar with the UI, here are some ways the project metadata panel is departing from the style of the rest of the UI.

1. Background color: The background color of the project metadata panel is a light gray, whereas the rest of the UI appears to use a white background. This color difference could make the panel feel disconnected from the rest of the UI.
2. Text styling: The text in the project metadata panel appears to be plain and without any special formatting or emphasis, whereas the rest of the UI appears to use different text styles such as bold, italic, and underlined to emphasize important information. This difference in text styling could make it more difficult for users to quickly scan and understand the information in the panel.
3. Layout: The layout of the project metadata panel is a grid-like structure with evenly spaced fields, whereas the rest of the UI appears to use a more fluid layout with varying content sizes and spacing. This difference in layout could make the panel appear more rigid and less visually interesting.
4. Edit buttons: These buttons are all aligned to the right side of the panel, and are placed in close proximity to each other.
The placement of these edit buttons could make the panel appear cluttered or visually overwhelming, especially if there are many other fields and elements in the panel. It could also make it more difficult for users to quickly locate and use the specific edit button they need.
5. Lack of hierarchy: Some of the elements in the UI could benefit from more clear visual hierarchy. For example, the project metadata panel appears to have several fields that are all similarly sized and spaced, which could make it difficult for users to quickly understand the most important information. Establishing a clearer hierarchy of information could help guide users' attention and make the UI easier to use.

**Comment by Lydiaofficial** [comment]
Hi @wetneb , could you review and give feedback on the first two tasks , please?
So I'd know if to go ahead with the design or make further research where necessary.

**Comment by wetneb** [comment]
@Lydiaofficial I agree with your points and I think they are actionable for a developer to implement this. Thanks a lot!

**Comment by Lydiaofficial** [comment]
Alright. Thank you.
